plugins/tactics/src/Ide/Plugin/Tactic/Context.hs

```diff
@@ -33,6 +36,9 @@ mkContext locals tcg = Context
   }
```

Comment:
Right now this function is one operation per line. The suggested changes would combine some of the lines, which seems worse.

plugins/tactics/src/Ide/Plugin/Tactic/Context.hs:(29,22)-(31,30): Suggestion: Use <=<
Found:
  (getFunBindId =<<) . fmap unLoc . bagToList
Perhaps:
  (getFunBindId Control.Monad.<=< (fmap unLoc . bagToList))

Reply:
Maybe
```
((getFunBindId . unLoc) <=< bagToList)
```
is better?
I'm not super inclined to this though...

Reply:
I could move these:
```
{- HLINT ignore mkContext "Redundant fmap" -}
{- HLINT ignore mkContext "Use <=<" -}
```

to the .hlint.yaml?

Reply:
I think, `Redundant fmap` rule may be useful for some case....

Reply:
I moved the fish recommendation ignore to the hlint.yaml and left the fmap recommendation in the module.